### ISSUE TITLE
'Add tree to collection' feature

### ISSUE THREAD

**Issue by jar398** [OpenTreeOfLife/opentree#584]
From study view:
    add preferred tree to... 
        new collection (create)
        existing collection
From tree (in study) view:
    add tree to...
         new collection
         existing collection

**Comment by kcranston** [comment]
The second of these is implemented in the tree popup window (although it is pretty buried and I worry that people won't discover it). The first is not yet implemented.

**Comment by jimallman** [comment]
I've added the "cherry-picking" UI to handle @jar398's second scenario. Note that I overlooked the study-level operation: "Add preferred tree(s) from this study to a new-or-existing collection."

I gather this would add **all** preferred trees (there can be more than one) without the user needing to approve each one. Do we want to pursue this for "collections v1"? If not, I'm inclined to close this issue or at least add the 'enhancements' tag for a second look later.

(As an aside, our wish list includes eventual storing of dynamic tree queries in a collection. Once we have this capability, it seems like this would be captured as such a query: _"Include any trees in this study that are marked as preferred. Add new preferred trees as they appear, and remove any that lose preferred status."_)
